src/debugSession/BrightScriptDebugSession.ts

```diff
@@ -1686,6 +1698,206 @@ export class BrightScriptDebugSession extends BaseDebugSession {
+                        let result = await this.rokuAdapter.getVariable(evalArgs.expression, args.frameId);
+                        parentVariable = await this.getVariableFromResult(result, args.frameId);
+                    } catch (error) {
+                        this.logger.error('Error looking up parent completions', error, { parentVariablePath });
+                    }
+                }
+
+                // provide completions for the parent variable if one was found
+                if (parentVariable) {
+                    let possibleFieldsAndMethods: AugmentedVariable[] = [];
+                    // Filter out virtual variables
+                    possibleFieldsAndMethods = parentVariable.childVariables.filter((v) => v.presentationHint?.kind !== 'virtual');
+
+                    for (let v of possibleFieldsAndMethods) {
+                        // Default completion type should be variable
+                        let completionType: DebugProtocol.CompletionItemType = 'variable';
+                        if (!supplyLocalScopeCompletions) {
+                            // We are not supplying local scope completions, so we need to determine the completion type relative to the parent variable
+                            if (parentVariable.type === 'roSGNode' || parentVariable.type === VariableType.AssociativeArray || parentVariable.type === VariableType.Object) {
+                                completionType = 'field';
+                            }
+
+                            switch (v.type) {
+                                case VariableType.Function:
+                                case VariableType.Subroutine:
+                                    completionType = 'method';
+                                    break;
+                                default:
+                                    break;
+                            }
+                        }
+
+                        completions.set(`${completionType}-${v.name}`, {
+                            label: v.name,
+                            type: completionType,
+                            sortText: '000000'
+                        });
+                    }
+
+                    let parentComponentType = this.debuggerVarTypeToRoType(parentVariable.type).toLowerCase();
+                    //assemble a list of all methods on the parent component
+                    const methods = [
+                        //if the parent variable is an actual interface (if applicable) Ex: `ifString` or `ifArray`
+                        ...interfaces[parentComponentType]?.methods ?? [],
+                        //interfaces from component of this name (if applicable) Ex: `roSGNode` or `roDateTime`
+                        ...components[parentComponentType]?.interfaces.map((i) => interfaces[i.name.toLowerCase()])?.methods ?? [],
```

Comment:
```suggestion
                        ...components[parentComponentType]?.interfaces.map((i) => interfaces[i.name.toLowerCase()]?.methods) ?? [],
```
We need to get the methods off the interface and return those.